INSTALL.md

```diff
@@ -237,7 +242,7 @@ Platform proto files.
 
 ```bash
 sudo zypper refresh && \
-sudo zypper install -y grpc-devel gzip libcurl-devel pkg-config tar wget
```

Comment:
Was removing `pkg-config` intentional here?

Reply:
It was removed in a previous PR so regenerating the INSTALL.md picked up those changes.